<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChewingEditor</class>
 <widget class="QMainWindow" name="ChewingEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chewing user phrase editor</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <widget class="QWidget" name="userphraseTab">
    <attribute name="title">
     <string>Userphrase</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="userphraseFilter"/>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="Notification" name="notification"/>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="UserphraseView" name="userphraseView">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="commitHistoryTab">
    <attribute name="title">
     <string>Commit History</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout2">
     <item row="0" column="0">
      <widget class="CommitHistoryView" name="commitHistoryView">
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_phrase"/>
    <addaction name="actionRemove_phrase"/>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Chewing_editor"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionAdd_phrase">
   <property name="text">
    <string>Add phrase</string>
   </property>
  </action>
  <action name="actionRemove_phrase">
   <property name="text">
    <string>Remove phrase</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionAbout_Chewing_editor">
   <property name="text">
    <string>About Chewing editor</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Notification</class>
   <extends>QLabel</extends>
   <header>Notification.h</header>
  </customwidget>
  <customwidget>
   <class>UserphraseView</class>
   <extends>QListView</extends>
   <header>UserphraseView.h</header>
  </customwidget>
  <customwidget>
   <class>CommitHistoryView</class>
   <extends>QListView</extends>
   <header>CommitHistoryView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
